web/modules/custom/portland_migrations/README.md

```diff
@@ -44,14 +44,14 @@ Long migrations run through terminus may exceed the Pantheon timeout and be term
 For example, the policies migration is lengthy due to custom processing and regularly times out. Use the following commands to reset and restart the migration:
 
 ```
-lando terminus remote:drush portlandor.powr-[ID] -- migrate:reset-status policies
```

Comment:
terminus documentation still says we need the "remote:" in front of Drush. Did you see something different?